docs/documentation/running_a_simulation/Running-a-Simulation.md

```diff
@@ -16,14 +16,23 @@ S_main_${TRICK_HOST_CPU}.exe [trick_version] [sie]
+- The first argument in the command line must be the simulation input file name. The input file name can be in the form of a full path name but MUST have a `RUN_<name>` directory immediately above the input file name. By default, all the simulation output is directed to this `RUN_<name>` directory. The standard `<input_file_name>` is input.py; however, a simulation could be started from a checkpoint file by substituting `chkpnt_<time>` in for `<input_file_name>` for non-Master/Slave and non-Import/Export simulations. For Master/Slave and Import/Export simulations you must have the simulation running, and the simulation must be in a freeze state before reloading a checkpoint.
 - The trick_version option will tell what version of Trick built the S_main executable.
-- The sie option will generate the smart input editor (SIE) resource file (CP will by default invoke the S_main executable with the sie option to generate this file).
-- The '-d' argument is optional and, if specified, starts the simulation in an input file verification mode. In this mode the entire input file is read, echoed to standard out, and then the simulation exits without calling any jobs listed in the S_define file. This mode helps debug input file syntax errors.
-- The '-O <output_file_path>' option allows the user to specify the directory to which simulation data log files will be written. If this option is omitted, the RUN_<name> directory is used.
-- The '-OO <output_file_path>' option allows the user to specify the directory to which ALL simulation output files will be written. If this option is omitted, the RUN_<name> directory is used.
-- The '--read-only-sim' flag can be used to redirect all files written at simulation runtime into the output directory.
-- The '-u' option specifies that all remaining arguments are meant to be used by user supplied jobs. All arguments after the -u can be accessed internal to the simulation jobs by calling the get_cmnd_args() function of the executive as illustrated below. In a master/slave simulation, the master's -u args will be passed to the slave.
+- The sie option will generate the smart input editor (SIE) resource file.
+- The `-d` argument is optional and, if specified, starts the simulation in an input file verification mode. In this mode the entire input file is read, echoed to standard out, and then the simulation exits without calling any jobs listed in the S_define file. This mode helps debug input file syntax errors.
+- The `-O <output_file_path>` option allows the user to specify the directory to which simulation data log files will be written. If this option is omitted, the `RUN_<name>` directory is used. 
```

Comment:
**ABSOLUTE CLARITY**
-O \<Output Directory\> = only RUN_* output goes into \<Output Directory\>
-OO \<Output Directory\> = ALL runtime output (RUN_*, DP_Product, copy of sie_resource) redirected to \<Output Directory\>
--read-only-sim = **REQUIRES -O OR -OO OUTPUT DIRECTORY** same as -OO.  If no -O or -OO output directory given, command line error with prejudice

if trick.sie_append_runtime_objs() is called from the input file:

-  with -O, sie_resource file in SIM directory is updated
-  with -OO or --real-time-sim, sie_resource in the -O/-OO directory is updated but NOT the one in the sim directory

Reply:
this is for john, he loves clarity

Reply:
Thank you! You will sleep better now.